testMVVM/EpisodeCreateViewController.swift

```diff
@@ -20,15 +29,14 @@ class EpisodeCreateViewController: UIViewController {
     }
 
     @IBAction func save() {
-
         _ = viewModel.save().then { _ -> Void in
-            self.dismiss(animated: true, completion: nil)
+            self.coordinator?.stop()
             }.catch { err in
```

Comment:
Just to let you know, if catch is implemented, then you don't need to discard the result. Also, the indentation is broken.